lib/types.ts

```diff
@@ -62,9 +62,10 @@ export enum ENVIRONMENT {
   SANDBOX = 'sandbox',
 }
 export enum REQUEST_SYNC_STRATEGIES {
-  SYNC_STRICT_TIMEOUT = 'sync-strict-timeout',
-  SYNC_SOFT_TIMEOUT = 'sync-soft-timeout',
+  SYNC = 'sync',
   ASYNC = 'async',
+  /** @deprecated Use {@link SYNC} and pass timeout on client initialization instead */
+  SYNC_STRICT_TIMEOUT = 'sync-strict-timeout',
```

Comment:
What's the thinking around keeping this (and not the other deprecated value) here and below?

Reply:
Better is still using this value, while literally no one is using the other one. The intent is to allow Better to upgrade, start using the new value and timeout, and then delete it in a later release. I suppose we could do it in one step, but this seemed more semver kosher.

Reply:
sg :v: